### PR TITLE
Jenkinsfile.merge

### PR DESCRIPTION
**Pull request by manics**
This is the equivalent of a devspace push job. Requires https://github.com/ome/build-infra/pull/1
Since there will be multiple Jenkinsfiles in this repo you must set the Jenkinsfile name in the push/merge job.

### CODE REVIEW

File: Jenkinsfile.merge [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+        BASE_REPO = 'omero-build.git'
+    }
+
+    stages {
+        stage('Merge') {
+            steps {
+                // build is in .gitignore so we can use it as a temp dir
+                sh 'mkdir build'
+                sh 'cd build && curl -sfL https://github.com/manics/build-infra/archive/omero-build.tar.gz | tar -zxvf -'
```
Comment (manics) [inline Jenkinsfile.merge]: This will need to be updated when https://github.com/ome/build-infra/pull/1 is merged

**Comment by manics** [comment]
Example job output
[OMERO-build-merge.txt](https://github.com/ome/omero-build/files/2967693/OMERO-build-merge.txt)

File: Jenkinsfile.merge (hunk 1 above)
Comment (joshmoore) [reply Jenkinsfile.merge]: good to go. also remove the commented out blocks while you're at it?

File: Jenkinsfile.merge [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    stages {
+        stage('Merge') {
+            steps {
+                // build is in .gitignore so we can use it as a temp dir
+                sh 'mkdir build'
+                sh 'cd build && curl -sfL https://github.com/ome/build-infra/archive/master.tar.gz | tar -zxf -'
+                sh 'virtualenv build/venv --system-site-packages && build/venv/bin/pip install scc'
+                sh """
+                    . build/venv/bin/activate
+                    if [ "${env.MERGE_PUSH_BRANCH}" != "null" ]; then
```
Comment (manics) [inline Jenkinsfile.merge]: I couldn't find a way to check for a non-existent environment variable. Setting the sheel to `bash` and using the `${VAR:-}` syntax doesn't work, testing for `"null"` is the only reliable method I've found so far.

**Comment by manics** [comment]
@joshmoore  Updated

**Comment by joshmoore** [comment]
Thanks, @manics. Generally looks good. Not sure why this is red though.

**Comment by manics** [comment]
It's passing after a restart

**Comment by joshmoore** [comment]
This will need merging for devspace `0.11`.

**Comment by joshmoore** [comment]
```
build/build-infra-master/recursive-merge: line 18: scc: command not found
```

**Comment by joshmoore** [comment]
Green build testing on "south" :+1:

**Pull request merged**